trigger.tex

```diff
@@ -69,6 +69,8 @@ \section{Actions}
 \hline
 2 -- 5 & Reserved for use by the trace specification. \\
 \hline
+8 -- 9 & Signal the firing of the trigger over an external debug trigger output to other blocks within the hart (e.g. as countable events to hpmcounters).  Generate a pulse on external debug trigger output 0 or 1 (respectively). \\
```

Comment:
I'd omit "over an external debug trigger output". Is it important that this is a pulse? That sounds like an implementation detail, but if there's already something elsewhere that says hpmcounters can have inputs that are pulsed then it would make sense to use the same definition here.

Reply:
I updated this.